int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -338,10 +341,14 @@ for (var i = 0; i < buttonSelector.length; i++) {
 if (window.location.href.toLowerCase().match(/checkout/)) {
 	setEdqInputSelectors("shipping");
 } else {
-	setEdqInputSelectors();
+	setEdqInputSelectors("");
 }
 if (edqEmailLineSelector) { edqEmailLineSelector.addEventListener("mouseover", function() {enableButtonDisable(edqCurrentSubmitButtonSelector, false);}); }
-if (edqPhoneLineSelectors) { edqPhoneLineSelectors.forEach(phoneSelector => { phoneSelector.addEventListener("mouseover", function() {enableButtonDisable(edqCurrentSubmitButtonSelector, false);}); }); }
+if (edqPhoneLineSelectors) {
+	for (var i = 0; i < edqPhoneLineSelectors.length; i++) {
+		edqPhoneLineSelectors[i].addEventListener("mouseover", function() {enableButtonDisable(edqCurrentSubmitButtonSelector, false);});
+	}
```

Comment:
Don't use C style loops here (especially since the logic you're implementing doesn't require a C style loop). In this particular case, since the issue is specific to a browser, let's use the [polyfill](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach).